src/app/components/HoldRequest/HoldRequest.jsx

```diff
@@ -18,7 +18,9 @@ class HoldRequest extends React.Component {
     super(props);
 
     this.state = _extend({
-      delivery: false,
+      delivery: (this.props.deliveryLocations.length > 0) ?
```

Comment:
Just check for `this.props.deliveryLocations.length` instead of `> 0` which are the same thing. Also, make a variable for `this.props.deliveryLocations.length` since you used it twice or for `this.props.deliveryLocations`.